kafka_consumer/check.py

```diff
@@ -157,18 +301,18 @@ def check(self, instance):
         zk_prefix = instance.get('zk_prefix', '')
 
         # If monitor_unlisted_consumer_groups is True, fetch all groups stored in ZK
+        consumer_groups = None
         if instance.get('monitor_unlisted_consumer_groups', False):
```

Comment:
This condition seems useless now that the default is `None` (see line above). We could merge it within the `elif`